pkgs/development/python-modules/deal-solver/default.nix

```diff
@@ -0,0 +1,72 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+
+, flit-core
+, z3
+, astroid
+
+  # tests
+, pytestCheckHook
+, hypothesis
+}:
+
+buildPythonPackage rec {
+  pname = "deal-solver";
+  version = "0.1.0";
+  format = "pyproject";
+  disabled = pythonOlder "3.6";
+
+  src = fetchFromGitHub {
+    owner = "life4";
+    repo = pname;
+    rev = "refs/tags/${version}";
```

Comment:
```suggestion
    rev = version;
```

Reply:
This is intentional and comming from [here](https://github.com/NixOS/nixpkgs/commit/7f5915b04801f383a3504a011601e2fb09dfedf8)